src/Main.java

```diff
@@ -1,23 +1,86 @@
-        EmployeeBook.employees[0] = new Employee("Васильев Иван Григорьевич", 2, 142_000);
-        EmployeeBook.employees[1] = new Employee("Алексеев Игорь Мстиславович", 1, 40_000);
-        EmployeeBook.employees[2] = new Employee("Антонов Алексей Георгиевич", 1, 41_000);
-        EmployeeBook.employees[3] = new Employee("Антипов Егор Николаевич", 3, 28_500);
-        EmployeeBook.employees[4] = new Employee("Васильев Платон Сергеевич", 2, 142_000);
-        EmployeeBook.employees[5] = new Employee("Сульянов Анатолий Султанович", 2, 112_000);
-        EmployeeBook.employees[6] = new Employee("Сахаров Захар Иннокентьевич", 5, 88_500);
-        EmployeeBook.employees[7] = new Employee("Землёв Никита Артемович", 4, 18_550);
-        EmployeeBook.employees[8] = new Employee("Вадиков Владимир Вадимович", 2, 109_520);
-        EmployeeBook.employees[9] = new Employee("Папазогло Сергей Петрович", 5, 23_770);
-
-    }
-       EmployeeBook.printEmployee(EmployeeBook.employees);
-       EmployeeBook.printListEmployee(EmployeeBook.employees);
-       EmployeeBook.totalSalary(EmployeeBook.employees);
-       EmployeeBook.minSalary(EmployeeBook.employees);
-       EmployeeBook.maxSalary(EmployeeBook.employees);
-       EmployeeBook.averageSalary(EmployeeBook.employees);
-
-}
+        for (int i = 0; i < EMPLOYEES.length; i++) {
+            EMPLOYEES[i] = generateEmployee();
+        }
+        System.out.println("Сумма ЗП: " + CalculateSumOfSalary());
+        System.out.println("Средняя ЗП: " + CalculateAverageOfSalary());
+        System.out.println("Сотрулник с максимальной ЗП: " + findMaxSalary());
+        System.out.println("Сотрулник с минимальной ЗП: " + findMinSalary());
+        printEmployees();
+        printFullNameOfEmployees();
+    }
+
+
+    private static void printEmployees() {
+        System.out.println("Сотрудники: ");
+        for (Employee employee : EMPLOYEES) {
+            System.out.println(employee);
+        }
+    }
+
+    private static void printFullNameOfEmployees() {
+        System.out.println("ФИО сотрудников: ");
+        for (Employee employee : EMPLOYEES) {
+            System.out.println(employee.getFullName());
+        }
+    }
+
+    private static int CalculateSumOfSalary() {
+        int sum = 0;
+        for (Employee employee : EMPLOYEES) {
+            sum += employee.getSalary();
+        }
+        return sum;
+    }
+
+    private static double CalculateAverageOfSalary() {
+        return (double) CalculateSumOfSalary() / EMPLOYEES.length;
+    }
+
+    private static Employee findMaxSalary() {
+        Employee employeeWithMaxSalary = null;
+        int maxSalary = Integer.MIN_VALUE;
+        for (Employee employee : EMPLOYEES) {
+            if (employee.getSalary() > maxSalary) {
+                maxSalary = employee.getSalary();
+                employeeWithMaxSalary = employee;
+            }
+        }
+        return employeeWithMaxSalary;
+    }
+        private static Employee findMinSalary() {
```

Comment:
```suggestion
        private static Employee findEmployeeWithMinSalary() {
```
это всё-таки сотрудник, а не ЗП